perllib/Open311/GetServiceRequestUpdates.pm

```diff
@@ -195,6 +195,8 @@ sub update_comments {
                     }
                 }
             }
+        } elsif (FixMyStreet->config('STAGING_SITE')) {
+            warn "Failed to match comment $request_id to problem for " . $body->name . "\n";
```

Comment:
This means we couldn't find a problem whose `external_id` is `$request_id`, could the warning clarify that? The current wording makes it seem like we're searching for a comment with `$request_id`.